### PR TITLE
[JENKINS-39979] Make sure that Branch and PR tabs show correct unsupp…

### PR DESCRIPTION
**Pull request by imeredith**
# Description
ATH tests for JENKINS-39979.

See [JENKINS-39979](https://issues.jenkins-ci.org/browse/JENKINS-39979).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees 

…orted box

### CODE REVIEW

**Comment by michaelneale** [comment]
Very good - thanks @imeredith  :bee: - merge this once https://github.com/jenkinsci/blueocean-plugin/pull/623 lands.

File: src/test/js/freestyle/branchPRTabs.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/** @module freestyle
```
Comment (scherler) [inline src/test/js/freestyle/branchPRTabs.js]: That is not correct, that should read ```branchTabs```

File: src/test/js/freestyle/branchPRTabs.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * @memberof karaoke
```
Comment (scherler) [inline src/test/js/freestyle/branchPRTabs.js]: That ```freestyle```

File: src/test/js/freestyle/branchPRTabs.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @description TEST: Branch and PR tabs show the correct display.
+ */
+module.exports = {
+/** Create freestyle Job "hijo"*/
```
Comment (scherler) [inline src/test/js/freestyle/branchPRTabs.js]: Are we creating a job called hijo?

**Pull request merged**